Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -148,10 +148,12 @@ void DoState(PointerWrap& p)
 static void ResetRegisters()
 {
   std::fill(std::begin(ppcState.ps), std::end(ppcState.ps), PairedSingle{});
-  std::fill(std::begin(ppcState.sr), std::end(ppcState.sr), 0U);
```

Comment:
This should be able to be 

```cpp
std::fill(std::begin(ppcState.sr), std::end(ppcState.sr), UReg_SR{});
```

no?

Reply:
I tried that, but the copy assignment operator for Bitfields is deleted, so a lot of errors get thrown.

Reply:
What about std::generate?

Reply:
This might be fixed if my suggestion for Common's BitField goes through (writing it up atm)

Reply:
Once PR #10847 is approved, I can use std::fill here once again.